apps/vaporgui/MainForm.cpp

```diff
@@ -1565,6 +1567,11 @@ void MainForm::redo()
 
 void MainForm::helpAbout()
 {
+    int nthreads = 1;
+#pragma omp parallel
+    {
+        nthreads = omp_get_num_threads();
+    }
```

Comment:
If there are more than 1 thread available, the variable `nthreads` will be written simultaneously by all threads, causing a data race condition. This variable should only be written to by the 1st thread.